.travis.yml

```diff
@@ -79,7 +79,7 @@ script:
 
 # Build documentation
 - cd $TRAVIS_BUILD_DIR/docs
-- hugo --verbose --environment staging
+- ../hugo --verbose --environment staging
```

Comment:
How about this?
```
- cd $TRAVIS_BUILD_DIR
- hugo --verbose --environment staging
```

Reply:
If I test that locally, I get 
```
Total in 0 ms
Error: Unable to locate config file or config directory. Perhaps you need to create a new site.
       Run `hugo help new` for details.
```

I can get around it by using the `--contentDir` and `--configDir` parameters, but then it gets hung up on a relative path:
```
$ hugo --verbose --environment staging --contentDir docs --configDir docs/config
Total in 0 ms
Error: module "learn" not found; either add it as a Hugo Module or store it in "/home/msalvatore/git/monkey/themes".: module does not exist
```